src/content/1.7/testing/how-to-create-your-own-behat-tests.md

```diff
@@ -0,0 +1,232 @@
+---
+title: How to create your own behat tests
+menuTitle: Creating your own behat tests
+weight: 20
+---
+
+# How to create your own behat tests or add tests to PrestaShop
+{{< minver v="1.7.6" title="true" >}}
+
+Behat tests are great if you want to validate the behavior of a component of the code and/or follow a user-oriented scenario.
+
+By "component" we usually mean a group of classes being called together, sometimes using an external dependdency (a web API, a database system like mysql ...).
+
+By "user-oriented scenario" we mean "a scenario that is a serie of steps, each step is an action, and at the end of the serie there is an expected result"
+
+## What is Behat
+
+[Behat][1] is a [Behavior-Driven Development](https://en.wikipedia.org/wiki/Behavior-driven_development) framework for PHP. 
+
+The main value of behat is that the test scenarios that are written are understandable by humans without technical knowledge needed. This makes them a lot more easier to read and maintain that some complex unit tests which are just code.
```

Comment:
```suggestion
Behat is a testing tool that brings one great asset to your tests: the test scenarios are written so that they are understandable by humans without technical knowledge needed ! This makes them easier to read and maintain.
```